compiler/src/dotty/tools/dotc/Driver.scala

```diff
@@ -32,34 +33,36 @@ class Driver {
   protected def emptyReporter: Reporter = new StoreReporter(null)
 
   protected def doCompile(compiler: Compiler, fileNames: List[String])(using Context): Reporter =
```

Comment:
Can we completely get rid of this `doCompile` and rename `doCompileFiles` into `doCompile`? Or at least make this one final so subclasses have to override the one with the implementation details and not the forwarder.

Reply:
done